tests/test_factory.py

```diff
@@ -887,6 +887,14 @@ def test_scalrecommendation():
     with pytest.raises(ValueError, match='"high" case not supplied'):
         pyscal_factory.create_scal_recommendation(baselow)
 
+    with pytest.raises(
+        ValueError, match="All values in parameter dict must be dictionaries"
+    ):
+
+        pyscal_factory.create_scal_recommendation(
+            {"low": [1, 2], "base": {"swl": 0.1}, "high": {"swl": 0.1}}
+        )
+
```

Comment:
Maybe belongs in a different PR, but I think it's ok to add a test

Reply:
borderline case, I didn't dare to fix the pylint issue without testing it..